apps/nextjs/src/components/AppComponents/Chat/chat-lessonPlanDisplay.tsx

```diff
@@ -73,8 +94,22 @@ export const LessonPlanDisplay = ({
   documentContainerRef: React.MutableRefObject<HTMLDivElement | null>;
   showLessonMobile: boolean;
 }) => {
+  const [lessonPlan, setLessonPlan] = useState<LooseLessonPlan>({});
+
   const chat = useLessonChat();
-  const { lessonPlan, ailaStreamingStatus, lastModeration } = chat;
+  const {
+    lessonPlan: incomingLessonPlan,
+    ailaStreamingStatus,
+    lastModeration,
+    streamingSections,
+  } = chat;
+
+  useEffect(() => {
+    if (Object.keys(incomingLessonPlan).length > 0) {
+      console.log("Update with incoming lesson plan", incomingLessonPlan);
```

Comment:
Just some logging - we could remove or have a flag? What's the status on having a logger for local dev?

Reply:
Have a look at #244!